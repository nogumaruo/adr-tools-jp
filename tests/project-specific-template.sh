adr init adrs
mkdir -p adrs/templates
cat > adrs/templates/template.md <<EOF
# {TITLE}

Project specific template!

# Status

{STATUS}

# Info

ADR Number: {NUMBER}

日付: {DATE}
決定者: @xxx, @xxx

EOF
adr new Aaa Bbb
adr new Ccc Ddd
cat adrs/002-aaa-bbb.md
cat adrs/003-ccc-ddd.md
