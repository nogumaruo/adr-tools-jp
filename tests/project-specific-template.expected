adr init adrs
adrs/001-record-architecture-decisions.md
mkdir -p adrs/templates
cat > adrs/templates/template.md <<EOF
# {TITLE}

Project specific template!

# Status

{STATUS}

# Info

ADR Number: {NUMBER}

日付: {DATE}
決定者: @xxx, @xxx

EOF
adr new Aaa Bbb
adrs/002-aaa-bbb.md
adr new Ccc Ddd
adrs/003-ccc-ddd.md
cat adrs/002-aaa-bbb.md
# Aaa Bbb

Project specific template!

# Status

提案中

# Info

ADR Number: 2

日付: 1992-01-12
決定者: @xxx, @xxx

cat adrs/003-ccc-ddd.md
# Ccc Ddd

Project specific template!

# Status

提案中

# Info

ADR Number: 3

日付: 1992-01-12
決定者: @xxx, @xxx

